RealmSwift/Tests/SwiftTestObjects.swift

```diff
@@ -78,6 +78,7 @@ class SwiftObject: Object {
 
     let arrayCol = List<SwiftBoolObject>()
     let setCol = MutableSet<SwiftBoolObject>()
+    let mapCol = Map<String, SwiftBoolObject>()
```

Comment:
This isn't a valid property declaration and should be throwing an exception complaining that object values need to be optional.

Reply:
Does this apply to embedded objects too?